substrait_consumer/producers.py

```diff
@@ -138,6 +144,83 @@ def name(self):
         return "IsthmusProducer"
 
 
+class DataFusionProducer:
```

Comment:
@richtia should we add short description about this producer in docstrings?

Reply:
Good point.  Looks like I missed adding them to all the producers.  I'll put something simple, like what we have in consumers.py